File: A - Imperative Basics/A01 - Hello World/A01.cs
//Author: Sady Sell Neto
//Copyright (C) 2016

using System;

//This is a C# line comment.
/*This is a C# block comment.*/

//Entry point in C#: Main method.
//It can have any access modifier and belong to a class, a struct or a
//non-static class. It must be static.
//Valid signatures:
//static void Main()
//static void Main(string[] args)
//static int Main()
//static int Main(string[] args)

//If you use the args parameter, you will be able to access command line
//options. Also, by returning an int instead of void, you can return an exit
//code.

class Program {

    public static void Main() {

        Console.WriteLine("Hello, world!");

    }
}

File: A - Imperative Basics/A01 - Hello World/A01.vb
' Author: Sady Sell Neto
' Copyright (C) 2016

Imports System

' This is a VB.NET line comment.
REM This is also a VB.NET line comment.
' VB.NET has no block comment.

' Entry point in VB.NET: Main method.
' It can have any access modifier and belong to a Class or a Structure or a
'   Module. It must be Shared (Module members are automatically Shared).
' Valid signatures:
' Sub Main()
' Sub Main(args As String())
' Function Main() As Integer
' Function Main(args As String()) As Integer

' If you use the args parameter, you will be able to access command line
'   options. Also, by using a Function instead of a Sub, you can return an exit
'   code.

Class Program

    Public Shared Sub Main()

        Console.WriteLine("Hello, world!")

    End Sub

End Class
